tests/unit/amo/pages/TestGuides.js

```diff
@@ -105,12 +112,11 @@ describe(__filename, () => {
       content.sections[0].header,
     );
 
-    const sectionExploreLink = root.find('.Guides-section-explore-more').at(0);
+    const sectionExploreMore = root.find('.Guides-section-explore-more').at(0);
+    const sectionExploreLink = sectionExploreMore.find(Link);
 
-    expect(sectionExploreLink).toHaveHTML(
-      oneLine`<div class="Guides-section-explore-more">Explore more
-      <a href="/en-US/android/collections/mozilla/password-managers/"
-      class="Guides-section-explore-more-link">password manager</a> staff picks.</div>`,
+    expect(sectionExploreLink.children().text()).toEqual(
```

Comment:
I think maybe you should be checking for `beforeLinkText` and `afterLinkText` as well.